### PR TITLE
Stats counters

### PR DESCRIPTION
**Pull request by blinktag**
* Renamed exiting "stats" package to "prometheus"
* Added stats counters
* Added `GetCurrentCounters()` and `GetCounterHistory()` endpoints

TODO: update plumber-schemas in go.mod once https://github.com/batchcorp/plumber-schemas/pull/32 is merged

### CODE REVIEW

File: prometheus/prometheus.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
+// Singleton so that it's easier to use in other packages
```
Comment (dselans) [inline prometheus/prometheus.go:1]: 👍

File: stats/stats.go [hunk 2]
```diff
@@ -1,122 +1,309 @@
...
+// AddCounter creates a new counter and launches a runFlusher() goroutine for it
+func (s *Stats) AddCounter(cfg *opts.Counter) *Counter {
+	// Check for existing counter
+	existing, _ := s.GetCounter(cfg.Type, cfg.Resource, cfg.ResourceId)
+	if existing != nil {
+		return existing
+	}
+
+	logger := logrus.WithFields(logrus.Fields{
+		"pkg":           "stats",
```
Comment (dselans) [inline stats/stats.go:138]: Include the counter friendly name in logs.

File: stats/stats_test.go [hunk 3]
```diff
@@ -0,0 +1,273 @@
+package stats
```
Comment (dselans) [inline stats/stats_test.go:1]: Really glad you included tests for this. TY!

**Review by dselans — APPROVED** [review]
Should try to include the friendly name (in both logs and return data) so it's a bit easier to interpret the metric data.

File: stats/stats.go (hunk 2 above)
Comment (blinktag) [reply stats/stats.go:158]: Fixed!

File: stats/stats_test.go (hunk 3 above)
Comment (blinktag) [reply stats/stats_test.go:1]: 🕺

**Pull request merged**